stepengine/src/main/java/org/tweetwallfx/devoxx18be/steps/mosaic/ImageMosaicAnimateImageStep.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2016-2018 TweetWallFX
```

Comment:
New class should only have 2018 in referenced years
```suggestion
 * Copyright 2018 TweetWallFX
```